simple-vm/terraform/terraform.tfvars

```diff
@@ -0,0 +1,7 @@
+project_id           = "home-258016"
+region               = "us-central1"
+zone                 = "us-central1-a"
+gce_ssh_user         = "demo"
+ssh_private_key_file = "id_ed25519"
+ssh_public_key_file  = "id_ed25519.pub"
+  
```

Comment:
Not sure why it says no new line at the EOF, probably cause you have an extra line?

Reply:
Fixed.